packages/task/src/browser/task-schema-updater.ts

```diff
@@ -55,13 +53,17 @@ export class TaskSchemaUpdater {
     protected readonly onDidChangeTaskSchemaEmitter = new Emitter<void>();
     readonly onDidChangeTaskSchema = this.onDidChangeTaskSchemaEmitter.event;
 
-    @postConstruct()
-    protected init(): void {
+    registerSchemas(context: JsonSchemaRegisterContext): void {
         const taskSchemaUri = new URI(taskSchemaId);
-        this.jsonSchemaStore.onDidChangeSchema(uri => {
-            if (uri.toString() === taskSchemaUri.toString()) {
+        const resource = this.inmemoryResources.add(taskSchemaUri, '');
+        if (resource.onDidChangeContents) {
+            resource.onDidChangeContents(() => {
                 this.onDidChangeTaskSchemaEmitter.fire(undefined);
-            }
+            });
+        }
+        context.registerSchema({
+            fileMatch: ['tasks.json', USER_TASKS_URI.toString()],
```

Comment:
@akosyakov when I'm trying to configure a global task, I get:
```
resource.ts:166 Uncaught (in promise) Error: A resource provider for 'user_storage:tasks.json' is not registered.
    at DefaultResourceProvider.<anonymous> (resource.ts:166)
    at step (resource.ts:15)
    at Object.throw (resource.ts:15)
    at rejected (resource.ts:15)
```

Reply:
@azatsarynnyy good catch! could you try again please?

Reply:
@akosyakov it's fixed now. Thanks!